acai-ios/BowlOrderDetailViewController.swift

```diff
@@ -8,60 +8,164 @@
+    var bottomFillerRect: UIView!
+    var collectionBottomFillerRect: UIView!
     var collectionView: UICollectionView!
     var listAdapter: ListAdapter!
+    var titleLabel: UILabel!
+    
+    // MARK: Gesture recognizers
+    var addToCartTapGestureRecognizer: UITapGestureRecognizer!
+    var backButtonTapGestureRecognizer: UITapGestureRecognizer!
     
     // MARK: Data
     var baseOptions: [OrderCustomizationOption]!
     var bowlItem: BowlItem!
     var toppingOptions: [OrderCustomizationOption]!
     
+    // MARK: Constraint Constants
+    let addToCartActionTabViewHeight = 55
+    let backButtonLeadingOffset = 18
+    let backButtonHeight = 15
+    let backButtonWidth = 8.7
+    let emptyItemHeight: CGFloat = 156
+    let titleLabelTopOffset = 5.5
+    
     // MARK: View Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .white
+        view.backgroundColor = .acaiColdGray
+//        navigationController?.navigationBar.shadowImage = UIImage()
+//        navigationController?.navigationBar.barTintColor = .clear
+//        navigationController?.navigationBar.tintColor = .white
+//        navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white, NSAttributedString.Key.font: UIFont.avenirNextBold.withSize(24)]
+//        navigationController?.navigationBar.backgroundColor = .clear
+        navigationController?.isNavigationBarHidden = true
         
         // hard code
         bowlItem = Acai.testBowl
         toppingOptions = bowlItem.toppingOptions
         baseOptions = bowlItem.baseOptions
-
+        
+        title = bowlItem.title
+        
+        titleLabel = UILabel()
+        titleLabel.font = UIFont.avenirNextBold.withSize(24)
+        titleLabel.textColor = .white
+        titleLabel.text = bowlItem.title
+        view.addSubview(titleLabel)
+        
+        titleLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(titleLabelTopOffset)
+        }
+        
+        backButtonImageView = UIImageView()
+        backButtonImageView.backgroundColor = .white
+        backButtonImageView.image = UIImage()
+        backButtonImageView.contentMode = .scaleAspectFit
+        backButtonImageView.isUserInteractionEnabled = true
+        view.addSubview(backButtonImageView)
+        
+        backButtonImageView.snp.makeConstraints { make in
+            make.centerY.equalTo(titleLabel.snp.centerY)
+            make.height.equalTo(backButtonHeight)
+            make.leading.equalToSuperview().offset(backButtonLeadingOffset)
+            make.width.equalTo(backButtonWidth)
+        }
+        
+        backButtonTapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(popBowlOrderDetailViewController))
+        backButtonImageView.addGestureRecognizer(backButtonTapGestureRecognizer)
```

Comment:
why are we using an `ImageView` + `UIGestureRecognizer` instead of just a `UIButton`?